src/main/apiClient.ts

```diff
@@ -183,46 +183,53 @@ export default class ApiClient {
   }
 
   loginFromSession(opts: AuthOptions = {}): Promise<void> {
-    if (!this.config.sso && !opts.idTokenHint) {
+    if (!this.config.sso && !opts.idTokenHint)
```

Comment:
```suggestion
    if (this.config.sso)
```
The previous condition is no longer true. We cannot call loginFromSession and/or checkSession using ID Tokens.